pkg/cluster/cluster.go

```diff
@@ -138,6 +138,16 @@ func (c *Cluster) prepareBackupAndRestore() error {
 		}
 	}
 
+	// We use backup RS to mark that we have done backup creation and even restore process before.
```

Comment:
I see.
My previous thought was that we could just save a round trip since it's unneeded.
Even better, avoiding unnecessary error.